### PR TITLE
feat: New .desktop shortcuts (Docs and Landing)

### PR DESCRIPTION
**Pull request by nicknamenamenick**
Using placeholder icons for this at least currently.

Bazzite Documentation opens https://docs.bazzite.gg.
Bazzite Landing opens https://universal-blue.discourse.group/docs?topic=8.

I really want to eventually add offline documentation, but for now this _may_ work.

### CODE REVIEW

File: system_files/desktop/shared/usr/share/applications/Bazzite Documentation.desktop [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+NoDisplay=false
+Terminal=false
+Exec=xdg-open https://docs.bazzite.gg
+Icon=/usr/share/ublue-os/bazzite/docs.svg
+Name=Bazzite Documentation
+Comment=Bazzite documentation
+Categories=Utility;
```
Comment (Malix-off) [inline system_files/desktop/shared/usr/share/applications/Bazzite Documentation.desktop:9]:
```suggestion
Categories=Utility;

```

File: system_files/desktop/shared/usr/share/applications/Bazzite Landing.desktop [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+NoDisplay=false
+Terminal=false
+Exec=xdg-open https://universal-blue.discourse.group/docs?topic=8
+Icon=/usr/share/ublue-os/bazzite/welcome.svg
+Name=Bazzite Landing
+Comment=Announcements, newsletters, and the forums
+Categories=Utility;
```
Comment (Malix-off) [inline system_files/desktop/shared/usr/share/applications/Bazzite Landing.desktop:9]:
```suggestion
Categories=Utility;

```

**Review by Malix-off — APPROVED** [review]
Add EOF

File: system_files/desktop/shared/usr/share/applications/Bazzite Landing.desktop (hunk 2 above)
Comment (nicknamenamenick) [reply system_files/desktop/shared/usr/share/applications/Bazzite Landing.desktop]: We might be doing this another way, but thanks for adding this fix if we end up going this route :smile:

**Pull request merged**